connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -363,6 +363,7 @@ private boolean sendRecords() {
             try {
                 maybeCreateTopic(record.topic());
                 final String topic = producerRecord.topic();
+                log.trace("{} is going to send record to {}", WorkerSourceTask.this, topic);
```

Comment:
If user opened the trace log, it'll see the producer is going to send record before producer.send got stuck